db-helpers.js

```diff
@@ -1,11 +1,41 @@
 const pg = require('pg')
 
+function setTimestampParsersToReturnStrings(types) {
+    // 1114 and 1184 are the OID for postgres TIMESTAMPTZ and TIMESTAMP
```

Comment:
Timestamps with postgres and the `node-pg` package are a bit 😢 
Postgres returns the timestamp (which could be UTC or any other timezone) to the Javascript package as a string. The package then converts it to a JavaScript `Date` obj, which means it is converted using the timezone of the JavaScript runtime. Probably not what anyone wants!

This hack prevents `node-pg` from converting the string value from postgres into anything but a string. We can then pass that string to a non-insane date library like moment.

Another pro tip is that Heroku's servers and postgres addon are all set to UTC (good!) but your own local postgres may not be.